cli/src/stake.rs

```diff
@@ -571,7 +571,11 @@ pub fn process_show_stake_account(
             }
             println!(
                 "stake activates starting from epoch: {}",
-                stake.activation_epoch
+                (if stake.activation_epoch < std::u64::MAX {
```

Comment:
Are the parenthesis necessary?   If not please remove them to reduce the syntax

Reply:
FYI, yeah, rust fmt said to write so... Anyway, it's no longer relevant as the code got simpler thanks to [this](https://github.com/solana-labs/solana/pull/7091#discussion_r349442345).